docs/src/content/docs/showcase.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Starlight Showcase
+description: See sites built with Starlight around the web!
+---
+Starlight is already being used in production around the web. See some examples below, and PR your own link if you have built your documentation site with Starlight!
+
+- [PubIndexAPI Docs](https://docs.pubindexapi.com/)
+- [har.fyi](https://har.fyi/)
+- [QBCore Docs](https://brycerussell.github.io/qbcore-docs/)
+- [Astro Error Pages](https://astro-error-page-documentation.vercel.app/)
+- [MES Docs](https://docs.muslimeng.org/)
```

Comment:
This one seems a bit WIP, so suggesting replacing with these two that are more polished:

```suggestion
- [Felicity](https://felicity.pages.dev/)
- [NgxEditor](https://sibiraj-s.github.io/ngx-editor/)
```

Going to suggest holding off merging this until we have #188 ready.